inference-engine/samples/benchmark_app/main.cpp

```diff
@@ -324,8 +330,37 @@ int main(int argc, char *argv[]) {
 
             slog::info << "Loading network files" << slog::endl;
 
+            CNNNetwork cnnNetwork;
+
             auto startTime = Time::now();
-            CNNNetwork cnnNetwork = ie.ReadNetwork(FLAGS_m);
+
+            if (FLAGS_mmap) {
+                // Read XML file content into a string
+                std::ifstream xml_file(FLAGS_m);
+                std::string xml_content((std::istreambuf_iterator<char>(xml_file)), std::istreambuf_iterator<char>());
+#if defined(ENABLE_UNICODE_PATH_SUPPORT) && defined(_WIN32)
+                std::wstring bin_file = std::wstring_convert<std::codecvt_utf8<wchar_t>>().from_bytes(FLAGS_m);
```

Comment:
Can we just have 2 methods to create mapped allocator:
```
std::shared_ptr<IAllocator> make_mmap_allocator(const std::string& path, size_t offset = 0, LockOp lock = LOCK_FOR_READ);
std::shared_ptr<IAllocator> make_mmap_allocator(const std::wstring& path, size_t offset = 0, LockOp lock = LOCK_FOR_READ);
```
And in this case we can use the first one in this sample (which simplifies the sample code)